turbopack/crates/turbo-tasks-backend/src/backend/mod.rs

```diff
@@ -90,15 +92,20 @@ pub enum TransientTask {
     Once(TransientTaskOnce),
 }
 
-pub struct TurboTasksBackend {
+pub struct TurboTasksBackend(Arc<TurboTasksBackendInner>);
+
+struct TurboTasksBackendInner {
```

Comment:
What's the motivation for making `Arc` internal here? Less calls to upgrade `Weak::upgrade` like in the memory backend?

Reply:
`spawn_blocking` requires a `'static` closure since the task can't be cancelled. So we need to use an Arc to clone the Arc into the closure.

Reply:
Oh, I was a little confused. [It's `turbo-tasks` that uses the `Weak::upgrade` trick](https://github.com/vercel/next.js/blob/ed10d58e23f101a161b337bbe88f364ef0800a2d/turbopack/crates/turbo-tasks/src/manager.rs#L361), not `turbo-tasks-memory`.

Eh, either way, this is fine.